rust/arrow/src/array/equal/mod.rs

```diff
@@ -841,6 +842,53 @@ mod tests {
         test_equal(a.as_ref(), b.as_ref(), true);
     }
 
+    #[test]
+    fn test_struct_equal_null() {
```

Comment:
Oops, this is what I used to test the null struct inheritance. So I'll remove it